templates/controller-deployment.yaml

```diff
@@ -58,6 +58,7 @@ spec:
         - |
           consul-k8s controller \
             --webhook-tls-cert-dir=/tmp/controller-webhook/certs \
+            --datacenter={{ .Values.global.datacenter }}
```

Comment:
does this take single `-`'s?

Reply:
I think they do. Want me to update all flags with a single `-`?

Reply:
I think that makes sense to do at some point

Reply:
how about now? 😉